cmd/dispenser/server.go

```diff
@@ -189,7 +189,7 @@ func dispense(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	tx, err := c.SendPaymentFromUnencryptedWallet(cfg.Source, target, uint64(cfg.Fee), uint64(cfg.Amount), nil)
+	tx, err := c.SendPaymentFromUnencryptedWallet(cfg.Source, target, uint64(cfg.Fee), uint64(cfg.Amount), [32]byte{}, nil)
```

Comment:
do we foresee `32` ever changing? (as well as its use in error checking in `clerk.go`)
if "no" this is fine, if "yes" it's probably still fine

Reply:
Hmm, probably in protocol upgrades only. Yeah, I'm considering having a typedef for it so that `32` can change all at once. Might go ahead and do this.